spec/models/location_spec.rb

```diff
@@ -10,6 +10,10 @@
     expect(Location.new(valid_attributes)).to be_valid
   end
 
+  it 'is invalid with invalid attributes' do
+    expect(Location.new(valid_attributes.without(:name))).to be_invalid
```

Comment:
Do you mind adding a quick check for location, too, while you're at it? If you do that, I'll merge this in. Thanks for the help!